5eShaped/translations/fr.json

```diff
@@ -529,10 +529,13 @@
   "PROFICIENCIES":"Maîtrises",
   "PROFICIENCY_BONUS":"Bonus de maîtrise",
   "PROFICIENT":"Maîtrise",
+  "PSIONICS":"psioniques",
   "PSYCHIC":"psychique",
   "PUBLIC_FOR_ALL":"Public",
   "QTY":"Qté",
   "QUERY":"Varie",
+  "QUICK":"Rapide",
```

Comment:
Will not work ![2016-10-24 22_43_18](https://cloud.githubusercontent.com/assets/14132657/19663214/5691b180-9a3b-11e6-993d-01fda054ccb7.png)
Probably use "Visible" until something better is found ![2016-10-24 22_44_53](https://cloud.githubusercontent.com/assets/14132657/19663255/8096b0ca-9a3b-11e6-8921-db883e54bce4.png)